.github/workflows/release.yml

```diff
@@ -66,33 +68,32 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - name: Install Packages
-        run: |
-          sudo apt install --yes jq
       - name: Package Pex ${{ needs.determine-tag.outputs.release-tag }} PEX
         uses: ./.github/actions/run-tox
         with:
           tox-env: package
       - name: Create ${{ needs.determine-tag.outputs.release-tag }} Release
-        run: |
-          RELEASE_TAG=${{ needs.determine-tag.outputs.release-tag }}
-          RELEASE_VERSION=${RELEASE_TAG#v}
-          curl \
-            --request POST \
-            --header "Accept: application/vnd.github.v3+json" \
-            --header "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
-            --url https://api.github.com/repos/pantsbuild/pex/releases \
-            --data '{
-              "tag_name": "'${RELEASE_TAG}'",
-              "name": "pex '${RELEASE_VERSION}'",
-              "body": "---\n\n## '${RELEASE_VERSION}'\n\nTODO: Add CHANGES.rst entries."
-            }' | tee response.json
-          echo "RELEASE_ID=$(jq '.id' response.json)" >> ${GITHUB_ENV}
+        id: create-release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ needs.determine-tag.outputs.release-tag }}
+          release_name: pex ${{ needs.determine-tag.outputs.release-version }}
+          body: |
+            ---
+
+            ## ${{ needs.determine-tag.outputs.release-version }}
+
+            TODO: Add CHANGES.rst entries.
+          draft: false
+          prerelease: false
       - name: Upload Pex ${{ needs.determine-tag.outputs.release-tag }} PEX
-        run: |
-          curl \
-            --request POST \
-            --header "Accept: application/vnd.github.v3+json" \
-            --header 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
-            --url https://api.github.com/repos/pantsbuild/pex/releases/${RELEASE_ID}/assets
```

Comment:
Aha, it was `s/api.github.com/uploads.github.com/`. But switching to canned actions is the way to go clearly anyhow.